src/middlewared/middlewared/plugins/pool.py

```diff
@@ -2025,8 +2025,10 @@ def import_disk_msdosfs_locales(self):
         await self.middleware.call('vm.stop_by_pool', pool['name'], True)
 
         job.set_progress(30, 'Stopping jails using this pool (if any)')
-        for jail_host in attachments['jails']:
-            await self.middleware.call('jail.stop', jail_host)
+        activated_pool = await self.middleware.call('jail.get_activated_pool')
+        if activated_pool == pool.name:
```

Comment:
Not sure if `pool.name` would work ? Pool would be a dict ?

Reply:
Whoops didn't notice this reverted that change, pushed a commit for it.